src/requestor.js

```diff
@@ -148,14 +152,21 @@ class LMRTFYRequestor extends FormApplication {
         }
         //console.log("LMRTFY socket send : ", socketData)
         if (saveAsMacro) {
+
             const actorTargets = actors.map(a => game.actors.get(a)).filter(a => a).map(a => a.name).join(", ");
             const user = game.users.get(formData.user) || null;
             const target = user ? user.name : actorTargets;
             const scriptContent = `// ${title} ${message ? " -- " + message : ""}\n` +
                 `// Request rolls from ${target}\n` +
-                `// Abilities: ${abilities.map(a => CONFIG.DND5E.abilities[a]).filter(s => s).join(", ")}\n` +
-                `// Saves: ${saves.map(a => CONFIG.DND5E.abilities[a]).filter(s => s).join(", ")}\n` +
-                `// Skills: ${skills.map(s => CONFIG.DND5E.skills[s]).filter(s => s).join(", ")}\n` +
+                (game.system.id = "pf2e" ?
+                    `// Abilities: ${abilities.map(a => CONFIG.PF2E.abilities[a]).filter(s => s).join(", ")}\n` +
+                    `// Saves: ${saves.map(a => CONFIG.PF2E.saves[a]).filter(s => s).join(", ")}\n` +
+                    `// Skills: ${skills.map(s => CONFIG.PF2E.skills[s]).filter(s => s).join(", ")}\n` 
+                    :
+                    `// Abilities: ${abilities.map(a => CONFIG.DND5E.abilities[a]).filter(s => s).join(", ")}\n` +
+                    `// Saves: ${saves.map(a => CONFIG.DND5E.abilities[a]).filter(s => s).join(", ")}\n` +
+                    `// Skills: ${skills.map(s => CONFIG.DND5E.skills[s]).filter(s => s).join(", ")}\n` 
+                ) + 
```

Comment:
Agreed with your statement, especially as it gets reused here, it makes it ugly to check here for pf2e/dnd5e so it would be better to save the abilities/saves/skills. Since both requestor and roller use it, I'd save them in the ready hook directory into the `LMRTFY` class, so you could do `LMRTFY.abilities` instead of `CONFIG.<system>.abilities` and it would simplify everything else in the code

Reply:
Resolved this, with caveat below in comments.